Code/Tools/FBuild/FBuildCore/FBuildOptions.h

```diff
@@ -70,6 +70,9 @@ class FBuildOptions
     uint32_t    m_CacheTrim                         = 0;
     int32_t     m_CacheCompressionLevel             = -1; // See Compresssor.h
 
+    // Worker Connection Limit
+    int         m_workerlimit                       = 15; 
```

Comment:
Can you please make this a uint32_t to adhere to the code style, as well as making it unsigned (since negative numbers make no sense in this case)

Can you also please rename the variable to "m_WorkerLimit" to adhere to the code style?

Reply:
Yes, I'll rename this variable.